servers.json

```diff
@@ -3,8 +3,7 @@
         "id": "dashpvp",
         "name": "DashPvP",
         "addresses": [
-            "play.dashpvp.net",
-            "dashpvp.net"
+            "play.dashpvp.net"
```

Comment:
No, keep dashpvp.net, remove `play.dashpvp.net`.

Reply:
dashpvp.net isnt a usable IP


> No, keep dashpvp.net, remove `play.dashpvp.net`.

Reply:
I know, but Lunar Client already checks for sub-domains, you don't need to specify it here.

Reply:
> I know, but Lunar Client already checks for sub-domains, you don't need to specify it here.

Done

Reply:
Thank you!